service/src/main/kotlin/app/cash/backfila/ui/pages/BackfillShowAction.kt

```diff
@@ -102,6 +103,24 @@ class BackfillShowAction @Inject constructor(
           }
         }
 
+        // State section with its own auto-reload
+        AutoReload(frameId = "backfill-$id-state") {
+          Card {
+            turbo_frame("backfill-$id-state") {
```

Comment:
AutoReload already includes a turbo_frame with this ID. I think this should be removed?